openai/logger.py

```diff
@@ -0,0 +1,277 @@
+try:
```

Comment:
should we name this module and class `wandb_logger` / `WandbLogger` respectively to make it more distinct from more generic logging lirbaries?